data/src/message.rs

```diff
@@ -88,6 +88,10 @@ impl Message {
         matches!(self.source, Source::Server)
     }
 
+    pub fn datetime(&self, fmt: &str) -> String {
+        self.datetime.with_timezone(&Local).format(fmt).to_string()
+    }
+
```

Comment:
We should either un-pub the field or name this differently from the field since one returns a DateTime and this one a string, but they're named the same.